src/async_local_storage.ts

```diff
@@ -0,0 +1,52 @@
+import * as semver from 'semver';
+import {Request, Response} from './functions';
+import {NextFunction} from 'express';
+import {requiredNodeJsVersionForLogExecutionID} from './options';
+
+export interface ExecutionContext {
+  executionId?: string;
+  traceId?: string;
+  spanId?: string;
+}
+
+let asyncLocalStorage: any;
```

Comment:
please fix: this should be probably be typed as `AsyncLocalStorage<ExecutionContext> | undefined`

Reply:
Oh you can import the type. Let me find a link.